cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -692,40 +770,24 @@
                             anchors.eq(anchors.length).focus();
                         }
                     }
-
-                    // hide subnav when hitting enter or escape
-                    if (e.keyCode === CMS.KEYS.ENTER || e.keyCode === CMS.KEYS.ESC) {
-                        nav.siblings('.cms-submenu-quicksearch').find('input').blur();
-                        CMS.Plugin._hideSubnav(nav);
-                    }
                 });
             },
 
             /**
-             * FIXME will only work with settings, add plugin will be handle differently
+             * Opens the settings menu for a plugin.
              *
-             * @param nav jQuery
-             * @param [dropdown=this.ui.dropdown] jQuery
              * @private
+             * @param nav {jQuery} trigger element
              */
-            _showSubnav: function (nav, dropdown) {
-                dropdown = dropdown || this.ui.dropdown;
-                nav.addClass('cms-btn-active');
-
-                var scrollHint = dropdown.find('.cms-submenu-scroll-hint');
-
+            _showSettingsMenu: function (nav) {
+                var dropdown = this.ui.dropdown;
                 var parents = nav.parentsUntil('.cms-dragarea').last();
+
+                nav.addClass('cms-btn-active');
                 parents.addClass('cms-z-index-9999');
-                // set visible states
```

Comment:
specifically add the removal to the description (PR) so we can make sure it gets noted on the release notes